proto/core/pointer/pointer.proto

```diff
@@ -13,4 +13,5 @@ message Pointer {
   repeated string tags = 5;
   string description = 6;
   string object_type = 7;
+  string attribute_name = 8;
```

Comment:
What is attribute name? Can't we deduce it from `points_to_object_with_path`?

Reply:
As I understand it, they are different. Let's use an example, 
```python
m = nn.Sequential()
m.add_module("fc1", nn.Linear(4,2))
m.add_module("fc2", nn.Linear(2,1))
m_ptr = m.send(duet)
```
Then, for `m_ptr`, the `points_to_object_with_path` of it would be like `torch.nn.Sequential`. Then, imagine if we want a pointer pointing to `m.fc1`, we can create a Pointer same as `m_ptr` but has `attribute_name=fc1`.